lib/graphql/plug/endpoint.ex

```diff
@@ -9,16 +9,18 @@ defmodule GraphQL.Plug.Endpoint do
       {mod, func} -> apply(mod, func, [])
       s -> s
     end
-    %{schema: schema}
+    root_value = Keyword.get(opts, :root_value, %{})
+    %{root_value: root_value, schema: schema}
```

Comment:
ordering? imagine schema is more important and should probably go first? was alpha ordering intentional here?